scripts/delete.sh

```diff
@@ -67,6 +67,11 @@ if [[ -n $path_to_configs ]]; then
 fi
 
 if [[ -n $namespace ]]; then
+  old_namespace=$(kubectl config  get-contexts $(kubectl config current-context) |tail -1|awk '{ print $5 }')
+  if [[ ! -n $old_namespace ]];then
+	old_namespace="default"
```

Comment:
Everything looks good. Just another nit. Consider to use the same format to be consistent with existing scripts.
1. add space before then
2. use 2 spaces as indentation instead of tab

Thanks!

Reply:
@chinhuang007 , Fixed, Thanks